### DISCUSSION TITLE
Tracking the roadmap - let's just a kanban board instead of https://flowershow.app/docs/roadmap

### DISCUSSION THREAD

**Discussion by rufuspollock** [flowershow/flowershow#183] [General and Help] [answered]
Currently we have https://flowershow.app/docs/roadmap plus some stuff listed on the front page.

Suggestion: converge on using a dedicated kanban board that only lists major features (the ones that matter for users). If wanted we can embed or link this from roadmap page.

Why:

* Currently we repeat information in ~3 places:
  * github issues
  * https://flowershow.app/docs/roadmap
  * A segment of front page
* This leads to things getting rapidly out of sync (as has already just happened as of 2022-09-23 where CLI tool is listing as coming soon but is in fact operational)
* Github issues tend to be the real source of truth. Using automation we can keep a kanban board up to date relatively easily

**Comment by olayway** [comment]
@rufuspollock agree, we don't need at least one of these 3 sources of information. I'd say the roadmap page should go and I'd stay with a few most important features we're working on on the home page.
When it comes to a dedicated kanban board: are you thinking about tagging issues that should go into it with e.g. "ux" or "ui" tags (or sth else)?

**Reply by rufuspollock** [reply]
> When it comes to a dedicated kanban board: are you thinking about tagging issues that should go into it with e.g. "ux" or "ui" tags (or sth else)?

- Not sure tagging matters so much - more just the major features and their status
- If we do want to categorise we already have "stream" for major categories app, publishing etc and could subdivide with ux/ui if needed

**Reply by olayway** [reply]
What I meant is how do you want to pull only specific issues into a dedicated kanban board, and my first thought was that we would pull some specific tags (i.e. categories for that). Anyways, I like the idea of having GH board as a single source of truth.

**Reply by rufuspollock** [reply]
Ok, i get it. that would make sense e.g. only "epic" items might go in or similar. We don't have that exactly at the moment and would have to think about it. For now i'll boot the board.

**Comment by rufuspollock** [comment, marked as answer]
AGREED: create a roadmap kanban and started here https://github.com/orgs/flowershow/projects/1/views/7?query=is%3Aopen+sort%3Aupdated-desc
